### PR TITLE
changed scipy installation from apt to pip.

### PR DESCRIPTION
**Pull request by dITySoftware**

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -92,6 +92,12 @@
     executable: pip3
     state: latest
 
+- name: Install python3-scipy
```
Comment (whotwagner) [inline tasks/main.yml:95]: do we have issues with scipy dpkg-packages?

Comment (dITySoftware) [reply tasks/main.yml:95]: There is an issue. ImportWarnings are occuring with the version in the apt repository. The pip version does not have any of these issues.

Comment (whotwagner) [reply tasks/main.yml:95]: I guess you created a symlink to the wrong location. please double check the symlink to the package

Comment (dITySoftware) [reply tasks/main.yml:95]: I think the python3-numpy installation is missing and the link is wrong because of that. I'm testing it.

Comment (dITySoftware) [reply tasks/main.yml:95]: I can now confirm, that numpy and scipy are located in /usr/lib/python3/dist-packages/.

The symlink is not wrong and by changing the installation from apt to pip the warning goes away. Currently the warning still occurs.

Comment (dITySoftware) [reply tasks/main.yml:95]: current apt version 1.3.3-3build1
current pip version 1.19.2

Comment (dITySoftware) [reply tasks/main.yml:95]: Please accept this PR or help me downloading a newer version with dpkg.

**Pull request closed without merging**